templates/tiddlywiki.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>tiddlywiki</Name>
+  <Repository>elasticdog/tiddlywiki</Repository>
+  <Registry>https://hub.docker.com/r/elasticdog/tiddlywiki/</Registry>
+  <Network>bridge</Network>
+  <Shell>sh</Shell>
+  <Privileged>false</Privileged>
+  <Support>https://hub.docker.com/r/elasticdog/tiddlywiki/</Support>
+  <Project>https://github.com/Jermolene/TiddlyWiki5</Project>
+  <Overview>TiddlyWiki is a self-contained JavaScript wiki, configured here to run as a Node.js application.&#xD;
+&#xD;
+Converted By Community Applications   Always verify this template (and values)  against the support page for the container&#xD;
+&#xD;
+</Overview>
+<WebUI>http://[IP]:[PORT:8080]</WebUI>
+  <Config Name="AppData Location" Target="/tiddlywiki" Default="/mnt/user/appdata/tiddlywiki" Mode="rw" Description="" Type="Path" Display="always" Required="false" Mask="false">/mnt/user/appdata/tiddlywiki</Config>
```

Comment:
```suggestion
  <Config Name="AppData Location" Target="/tiddlywiki" Default="/mnt/user/appdata/tiddlywiki" Mode="rw" Description="" Type="Path" Display="always" Required="true" Mask="false">/mnt/user/appdata/tiddlywiki</Config>
```

Let's avoid in-container-only data, could inadvertently lead to users bloating their `docker.img`